cluster_orchestrator/cluster-manager/cluster_manager.py

```diff
@@ -184,76 +188,8 @@ def test_disconnect():
     app.logger.info("Websocket - Client disconnected")
 
 
-# ........... BEGIN register to System Manager .........#
-# ......................................................#
-
-
-@sio.on("sc1", namespace="/register")
-def handle_init_greeting(jsonarg):
-    app.logger.info("Websocket - received System_Manager_to_Cluster_Manager_1 : " + str(jsonarg))
-    data = {
-        "manager_port": MY_PORT,
-        "network_component_port": NETWORK_COMPONENT_PORT,
-        "cluster_name": MY_CHOSEN_CLUSTER_NAME,
-        "cluster_info": {},
-        "cluster_location": MY_CLUSTER_LOCATION,
-    }
-    time.sleep(1)  # Wait to Avoid Race Condition!
-
-    sio.emit("cs1", data, namespace="/register")
-    app.logger.info("Websocket - Cluster Info sent. (Cluster_Manager_to_System_Manager)")
-
-
-@sio.on("sc2", namespace="/register")
-def handle_init_final(jsonarg):
-    app.logger.info("Websocket - received System_Manager_to_Cluster_Manager_2:" + str(jsonarg))
-    data = json.loads(jsonarg)
-
-    app.logger.info("My received ID is: {}\n\n\n".format(data["id"]))
-
-    global MY_ASSIGNED_CLUSTER_ID
-    MY_ASSIGNED_CLUSTER_ID = data["id"]
-
-    sio.disconnect()
-    if MY_ASSIGNED_CLUSTER_ID is not None:
-        app.logger.info("Received ID. Go ahead with Background Jobs")
-        prometheus_init_gauge_metrics(MY_ASSIGNED_CLUSTER_ID, app.logger)
-        background_job_send_aggregated_information_to_sm()
-    else:
-        app.logger.info("No ID received.")
-
-
-@sio.event()
-def connect():
-    app.logger.info("Websocket - I'm connected to System_Manager!")
-
-
-@sio.event()
-def connect_error(m):
-    app.logger.info("Websocket connection failed with System_Manager!")
-
-
-@sio.event()
-def error(sid, data):
-    app.logger.info(">>>> Websocket error with System_Manager <<<<< ")
-
-
-@sio.event()
-def disconnect(m):
-    app.logger.info("Websocket disconnected with SM!")
-
-
-def init_cm_to_sm():
-    app.logger.info("Connecting to System_Manager...")
-    try:
-        sio.connect(SYSTEM_MANAGER_ADDR + "/register", namespaces=["/register"])
-    except Exception:
-        app.logger.error("SocketIO - Connection Establishment with System Manager failed!")
-    time.sleep(1)
-
-
-# ......... FINISH - register at System Manager ........#
-# ......................................................#
+# ...... Websocket END Handling with edge nodes .......#
```

Comment:
We can safely remove websocket based registration to worker nodes as well. That's deprecated since we migrated to go-based node engines. Can you erase that? 🙏🏻

Reply:
Sure! Let me just test if anything breaks as I already tried to remove legacy part and resulted in inconsistent behavior 👍